README.md

```diff
@@ -355,6 +356,75 @@ writable.
 
 Always prefer `ReadonlyArray` over a regular `Array` unless it must be possible to modify the Array.
 
+### React props & TypeScript
+
+#### Why use Interface instead of Type for typing props?
+
+`interface` are a little more powerful than the corresponding `type` declaration.
+
+One of the reasons to use `interface` is when you need to compose two or more types, you have the
+option of extending those types with an interface, or intersecting them in a type alias. TS team
+says:
+
+> Interfaces create a single flat object type that detects property conflicts, which are usually
+> important to resolve! Intersections on the other hand just recursively merge properties, and in
+> some cases produce never. Interfaces also display consistently better, whereas type aliases to
+> intersections can't be displayed in part of other intersections. Type relationships between
+> interfaces are also cached, as opposed to intersection types as a whole. A final noteworthy
+> difference is that when checking against a target intersection type, every constituent is checked
+> before checking against the "effective"/"flattened" type. For this reason, extending types with
+> interfaces/extends is suggested over creating intersection types.
```

Comment:
I think linking to the page & heading where the TS team are quoting this would be nice.

Reply:
Done ✅